modules/admin-top-bar/assets/scss/admin.scss

```diff
@@ -0,0 +1,211 @@
+@import "../../../../assets/dev/scss/editor/breakpoints";
+
+$top-bar-height: 50px;
+
+// Hide the widget in admin dashboard that was created by checking the screen option checkbox.
+// "display: none" won't work here since it will influence the mechanism for saving the state for the checkbox in screen options
+#e-dashboard-widget-admin-top-bar {
+	position: absolute;
+	opacity: 0;
+	pointer-events: none;
+}
+
+#e-admin-top-bar {
+	@include end( 0 );
+	background: $white;
+	box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.0394871);
+	display: none;
+	position: absolute;
+	top: 0;
+	width: calc(100% - 160px);
+	z-index: $first-layer;
+
+
+	.e-admin-top-bar {
+		display: flex;
+		height: $top-bar-height;
+		justify-content: space-between;
+		padding: 0 16px;
+	}
+
+	.page-title-action {
+		background: $gray-lightest;
+		border: 1px solid $blue;
+		border-radius: 3px;
+		box-sizing: border-box;
+		color: $blue;
+		cursor: pointer;
+		display: inline-block;
+		line-height: 30px;
+		margin: 0 4px;
+		min-height: 30px;
+		padding: 0 10px;
+		text-decoration: none;
+
+		&:hover {
+			background: $gray-lighter;
+			border-color: $blue-flat;
+			color: $blue-flat;
+		}
+	}
+
+	.e-admin-top-bar__heading {
+		display: inline-flex;
+		align-items: center;
+		justify-content: center;
+		margin-inline-end: 40px;
+	}
+
+	.e-admin-top-bar__main-area {
+		display: inline-flex;
+		align-items: center;
+		justify-content: center;
+
+		button {
+			margin: 0 4px
+		}
+	}
+
+	.e-admin-top-bar__secondary-area {
+		display: inline-flex;
+		align-items: center;
+		justify-content: center;
+
+	}
+
+	.e-admin-top-bar__heading-logo {
+		width: 30px;
+	}
+
+	.e-admin-top-bar__heading-logo {
+		background: $editor-accent;
+		border-radius: 15px;
+		color: $white;
+		display: inline-block;
+		font-size: pxToRem(14px);
+		height: pxToRem(30px);
+		line-height: pxToRem(30px);
+		text-align: center;
+		width: pxToRem(30px);
+	}
+
+	.e-admin-top-bar__heading-title {
+		color: $editor-darkest;
+		font-size: 15px;
+		font-weight: 700;
+		font-family: 'Roboto';
+		padding: 0 8px;
+		line-height: normal;
+		text-transform: uppercase;
+	}
+
+	&.top-bar-active {
+		display: block;
+
+		&~ #wpbody {
+			#wpbody-content {
+				margin-top: $top-bar-height;
+			}
+
+			.wrap h1 {
+				display: none;
+			}
+		}
+	}
+
+	&:not(.top-bar-active ) ~ #wpbody {
+		.wrap h1 {
+			display: block;
+		}
+	}
+
+	.e-admin-top-bar__bar-button {
+		align-items: center;
+		cursor: pointer;
+		display: inline-flex;
+		justify-content: center;
+		margin: 0 10px;
+		text-decoration: none;
+		color: $editor-dark;
+
+		.e-admin-top-bar__bar-button-title {
+			color: $editor-dark;
+		}
+
+		.e-admin-top-bar__bar-button-icon {
+			margin: 0 4px;
+		}
+
+		&:hover {
+			.e-admin-top-bar__bar-button-title,
+			.e-admin-top-bar__bar-button-icon{
+				color: $editor-accent;
+			}
+		}
+	}
+
+	.e-admin-top-bar__bar-button-title {
+		font-size: 13px;
+		font-weight: 500;
+		font-family: 'Roboto';
```

Comment:
Where is the font enqueue? How should it work for other languages?

Reply:
Done.I have added the fonts in enqueue_scripts.